modules/programs/foot.nix

```
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.foot;
+  iniFormat = pkgs.formats.ini { };
+in {
+  meta.maintainers = with lib.maintainers; [ plabadens ];
+
+  options.programs.foot = {
+    enable = mkEnableOption "Foot terminal";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.foot;
+      defaultText = literalExample "pkgs.foot";
+      description = "The foot package to install";
+    };
+
+    server.enable = mkEnableOption "Foot terminal server";
+
+    settings = mkOption {
+      type = iniFormat.type;
+      default = { };
+      description = ''
+        Configuration written to
+        <filename>~/.config/foot/foot.ini</filename>. See <link
+        xlink:href="https://codeberg.org/dnkl/foot/src/branch/master/foot.ini"/>
+        for a list of available options.
+      '';
+      example = literalExample ''
+        {
+          main = {
+            term = "xterm-256color";
+
+            font = "Fira Code:size=11";
+            dpi-aware = "yes";
+          };
+
+          mouse = {
+            hide-when-typing = "yes";
+          };
+        }
+      '';
+    };
+  };
+
+  config = mkMerge [
+    (mkIf cfg.enable {
```

No need to use `mkMerge` here
```suggestion
  config = mkIf cfg.enable {
```